VimeoNetworking/VimeoNetworking/ErrorCode.swift

```diff
@@ -39,11 +39,11 @@ public enum VimeoErrorCode: Int
     case FacebookJoinMalformedToken = 2305
     case FacebookJoinDecryptFail = 2307
     case FacebookJoinTokenTooLong = 2308
-    case FacebookLogInNoToken = 2312
-    case FacebookLogInMissingProperty = 2310
-    case FacebookLogInMalformedToken = 2311
-    case FacebookLogInDecryptFail = 2313
-    case FacebookLogInTokenTooLong = 2314
```

Comment:
Nit pick but we make "Log In" two words everywhere in the app. I think it makes sense to keep it that way here.